Production/crab/configs/Phase2_110X/Zprime.txt

```diff
@@ -0,0 +1,4 @@
+Automatic
+globalTag=110X_mcRun4_realistic_v3 sampleType=MC_Phase2_110X
+
+ZprimeToMuMu_M-6000_PU140 1000 /ZprimeToMuMu_M-6000_TuneCP5_14TeV-pythia8/Phase2HLTTDRWinter20RECOMiniAOD-PU140_110X_mcRun4_realistic_v3-v2/MINIAODSIM
```

Comment:
There is no Z'->tautau or tau gun samples? Otherwise, the statistics for real tau with high pt will be very limited.

Reply:
Nope, see [here](https://cmsweb.cern.ch/das/request?view=list&limit=50&instance=prod%2Fglobal&input=dataset+status%3D*+dataset%3D%2FZprime*%2F*Run4*%2FMINIAODSIM)

Reply:
I'll have a look for TauGun, though

Reply:
Here is the full list of the considered [Phase2 campaign](https://cmsweb.cern.ch/das/request?view=list&limit=150&instance=prod%2Fglobal&input=dataset+status%3D*+dataset%3D%2F*%2FPhase2*Run4*%2FMINIAODSIM)

The only ones that seem to be fine are [MultiTau samples](https://cmsweb.cern.ch/das/request?view=list&limit=150&instance=prod%2Fglobal&input=dataset+status%3D*+dataset%3D%2FMultiTau*%2FPhase2*Run4*%2FMINIAODSIM)

Reply:
I think it should be fine though just to request the samples from the Tau POG side

Reply:
Yes, I think multi-tau samples should be for the training.
N.B. maybe in the future we'll need to add non-overlap criterion for shuffle&merge step: e.g. exclude taus if their isolation cones overlap from the default training.

Reply:
Added MultiTau samples